template/templates.yml

```diff
@@ -0,0 +1,30 @@
+---
+# Templates for Spice Instant Answers
+#
+# Format:
+#   template_name:
+#       template_source_file:
+#           output: output_file # This is a Text::Xslate template itself
+#           group:  group_name  # Human readable group that this file belongs to
+#           optional: 1         # boolean; indicates if this file is optional
+#   
+#   template_name2:
+#       ...
+
+default:
+    lib/DDG/Spice/Example.pm:
+        output: lib/DDG/Spice/<:$ia_package_name:>.pm
+        group:  code
+
+    t/Example.t:
+        output: t/<:$ia_package_name:>.t
+        group:  test
+
+    share/spice/example/example.handlebars:
+        output:   share/spice/<:$lia_name:>.handlebars
+        group:    handlebars
+
+    share/spice/example/example.js:
+        output:   share/spice/<:$lia_name:>.js
```

Comment:
This should be in a sub-directory, and needs to be able to handle further subdirectories -- I can't remember if that's builtin already?

e.g:

- Spice::Movie => /share/spice/movie/movie.js
- Spice::Movie::Showtimes => /share/spice/movie/showtimes/movies_showtimes.js

The same applies to all files in share directories